docs/_frontend/src/models/SearchCombobox/SearchCombobox.js

```diff
@@ -0,0 +1,181 @@
+import classNames from 'classnames';
+import { useCombobox, useMultipleSelection } from 'downshift';
+import React, { useState } from 'react';
+import debounce from 'lodash.debounce';
+import styles from './SearchCombobox.module.css';
+
+const SearchCombobox = ({
```

Comment:
Clicking on the arrow button in the dropdown loads 25000 records. We should not load anything. For example, we could display something like "Start typing..."

Reply:
Added the `Start typing ...` text for empty input values